pkg/shell/shell.css

```diff
@@ -627,6 +627,16 @@ ul.available-interfaces-group {
     vertical-align: top;
 }
 
+/* workaround: make text red for select if contained in has-error, not in patternfly */
+.has-error > .bootstrap-select > button > span {
+    color: #A94442;
+}
+
+
+.containers-run-message {
```

Comment:
> Did you try it? An element like this gets display: none; for me.
> 

According to [this page](http://www.w3schools.com/tags/att_global_hidden.asp), using hidden would push our browser compatibility from IE10 to IE11 as a minimum version...